docusaurus/docs/learn/quickstarts/network/local-no-docker.md

```diff
@@ -58,16 +58,22 @@ ziti-controller started as process id: 1286. log located at: /home/vagrant/.ziti
 
 ### Verify the Controller is Running
 
-Assuming you have sourced the script, you will have an environment variable set named `$ZITI_EDGE_CONTROLLER_API`. After
-the controller has started, your controller should be listening at that hostname:port combination. You can see what your
-value is set to by running `echo $ZITI_EDGE_CTRL_ADVERTISED`. This variable defaults to: `$(hostname):1280`. Make sure the
-controller is on and listening and then start the edge router. 
+Assuming you have sourced the OpenZiti quickstart script, you will have environment variables set named 

```

Comment:
> OpenZiti quickstart script

maybe also link to it? or something like that?

Reply:
also technically it's probably: 'assuming you ran the expressInstall function' too ... just sourcing the script won' tdo it?